tests/virt_autotest/login_console.pm

```diff
@@ -26,6 +26,8 @@ sub login_to_console {
     reset_consoles;
     select_console 'sol', await_console => 0;
 
+    my $sut_machine = get_var('SUT_IP', 'nosutip');
```

Comment:
Use ```get_required_var``` instead and also, mind renaming it to `$hostname`  or `$sut_hostname`? just for semantics as that's what we use somewhere else...

```suggestion
    my $sut_machine = get_required_var('SUT_IP');
```

Reply:
I was not sure whether SUT_IP always exists in all test suites, so I used get_var in order to avoid exception. If you think get_required_var is more suitable, I will make the change.

Reply:
@waynechen55 you are right, better keep it to get_var. Or explore the possibility to set in the machine definition, something like? `SUT_VENDOR=foo` so that you can just simply do `is_remote_backend && check_var('SUT_VENDOR', 'foo')`? wdyt?